pkgs/development/compilers/elm/packages/lamdera.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv, lib
+, fetchurl
+, autoPatchelfHook
+, gmp5, ncurses5, zlib
+}:
+
+stdenv.mkDerivation rec {
+  pname = "lamdera";
+  version = "1.0.1";
+
+  src = fetchurl {
+    url = "https://static.lamdera.com/bin/linux/lamdera-v${version}";
+    sha256 = "15dee9df5d4e71b07a65fbd89d0f7dcd8c3e7ba05fe2b0e7a30d29bbd1239d9f";
+  };
+
+  dontUnpack = true;
+
+  nativeBuildInputs = [
+    autoPatchelfHook
+  ];
+
+  buildInputs = [
+    gmp5
+    ncurses5
+    zlib
+  ];
+
+
+  installPhase = ''
+    install -m755 -D $src $out/bin/lamdera
+  '';
+
+  meta = with lib; {
+    homepage = https://lamdera.com;
```

Comment:
this breaks nixpkgs.tarball on unstable-small, because of the url literal

Reply:
should we be using strings @ajs124 ?

Reply:
fix https://github.com/NixOS/nixpkgs/pull/189019

Reply:
Sorry, I wrote the initial version of this derivation long before I was aware that URL literals are discouraged. It would be good to get a CI linter for this.

Reply:
Adding this to ofborg was attempted in https://github.com/NixOS/ofborg/pull/597 but ultimately reverted